src/components/Comments/CommentCard/CommentCard.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import { shallow } from "enzyme";
+import React from "react";
+import CommentCard from "./CommentCard";
+
+const props = {
+  onChange: () => {},
+  onSubmitReply: jest.fn()
+};
+const mockCommentCard = jest.fn();
+let wrapper;
+describe("CommentCard test", () => {
+  beforeEach(() => {
+    wrapper = shallow(<CommentCard {...props} />);
+  });
+
+  it("matches snapshot", () => {
+    const component = shallow(<CommentCard {...mockCommentCard} />);
+    expect(component).toMatchSnapshot();
+  });
+  it("should render a form", () => {
+    expect(wrapper.find("form")).toHaveLength(1);
+  });
+
+  it("should simulate onChange event", () => {
+    wrapper.find({ id: "reply" }).simulate("change", {
+      target: { name: "comment_body", value: "Hello kasule" }
+    });
+    expect(wrapper.find("#reply").length).toEqual(1);
```

Comment:
This assertion is not testing a relevant case. It should check the value of `#reply` input field matches the 'value' passed in the target instead of the length